### PR TITLE
Added the App Manager, ProtocolERC20 and ProtocolERC721  documents

### PR DESCRIPTION
**Pull request by mpetersoCode55**
Added the App Manager, ProtocolERC20 and ProtocolERC721  documents and corrected API consistency issue for Access Levels.

### CODE REVIEW

File: docs/userGuides/Architecture/Cient/APP-MANAGER.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
+# App Manager
```
Comment (TJ-Everett) [inline docs/userGuides/Architecture/Cient/APP-MANAGER.md]: I cannot comment on the directory files so I'll drop it here but the folder name is misspelled. Cient vs Client

File: docs/userGuides/Architecture/Cient/APP-MANAGER.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+- Risk Scores
+- Access Levels
+
+The App Manager also provides the ability to check application level rules via its associated Application Handler.
+
+### Metadata
+
+#### Roles
+
+The App Manager can be utilized for both updating and checking an accounts role in relation to the application. These capabilites are provided for the following roles:
```
Comment (TJ-Everett) [inline docs/userGuides/Architecture/Cient/APP-MANAGER.md]: accounts should have an ' -> account's

File: docs/userGuides/Architecture/Cient/PROTOCOL-ERC-20.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+function pause() public virtual appAdministratorOnly(appManagerAddress)
+```
+
+- unpause: overridden to apply additional caller constraints (can only be called by an App Admin).
+
+```c
+function unpause() public virtual appAdministratorOnly(appManagerAddress) 
+```
+
+- _beforeTokenTransfer: overridden to add check the protocol rules first.
```
Comment (TJ-Everett) [inline docs/userGuides/Architecture/Cient/PROTOCOL-ERC-20.md]: change "overridden to add check the protocol rules first." to "overridden to allow protocol rule check hook. "

File: docs/userGuides/Architecture/Cient/PROTOCOL-ERC-20.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+```
+
+- flashLoan: overridden to explicitly not allow flashloans.
+
+```c
+function flashLoan(IERC3156FlashBorrower receiver, address token, uint256 amount, bytes calldata data) public pure virtual override returns (bool)
+```
+
+### Added Functions
+The following functions have been added specific to the ProtocolERC20 contract:
```
Comment (TJ-Everett) [inline docs/userGuides/Architecture/Cient/PROTOCOL-ERC-20.md]: specifically*

File: docs/userGuides/Architecture/Cient/PROTOCOL-ERC-721.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+function unpause() public virtual appAdministratorOnly(appManagerAddress)
+```
+
+- safeMint: overridden to increment the internal tokenIdCounter, only allow when not paused and to apply additional caller constraints (can only be called by an App Admin).
+
+```c
+function safeMint(address to) public payable virtual whenNotPaused appAdministratorOrOwnerOnly(appManagerAddress)
+```
+
+- _beforeTokenTransfer: overridden to add check the protocol rules first.
```
Comment (TJ-Everett) [inline docs/userGuides/Architecture/Cient/PROTOCOL-ERC-721.md]: same as above for rephrasing

File: docs/userGuides/Architecture/Cient/PROTOCOL-ERC-721.md [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+function getHandlerAddress() external view override returns (address)
+```
+
+- setBaseURI: used to set the URI for the contract.
+
+```c
+function setBaseURI(string memory _baseUri) public virtual appAdministratorOnly(appManagerAddress)
+```
+
+- _baseURI: used to retreive the URI for the contract.
```
Comment (TJ-Everett) [inline docs/userGuides/Architecture/Cient/PROTOCOL-ERC-721.md]: retreive - > retrieve

File: docs/userGuides/Architecture/Cient/APP-MANAGER.md (hunk 1 above)
Comment (mpetersoCode55) [reply docs/userGuides/Architecture/Cient/APP-MANAGER.md]: well that's embarrassing. Resolved!

**Review by TJ-Everett — CHANGES_REQUESTED** [review]
Looks great overall Michael. The folder name change and some rephrasing suggestions and this thing will be done!

File: docs/userGuides/Architecture/Cient/APP-MANAGER.md (hunk 2 above)
Comment (mpetersoCode55) [reply docs/userGuides/Architecture/Cient/APP-MANAGER.md]: Resolved.

File: docs/userGuides/Architecture/Cient/PROTOCOL-ERC-20.md (hunk 3 above)
Comment (mpetersoCode55) [reply docs/userGuides/Architecture/Cient/PROTOCOL-ERC-20.md]: Resolved.

File: docs/userGuides/Architecture/Cient/PROTOCOL-ERC-20.md (hunk 4 above)
Comment (mpetersoCode55) [reply docs/userGuides/Architecture/Cient/PROTOCOL-ERC-20.md]: Resolved

File: docs/userGuides/Architecture/Cient/PROTOCOL-ERC-721.md (hunk 5 above)
Comment (mpetersoCode55) [reply docs/userGuides/Architecture/Cient/PROTOCOL-ERC-721.md]: Resolved

File: docs/userGuides/Architecture/Cient/PROTOCOL-ERC-721.md (hunk 6 above)
Comment (mpetersoCode55) [reply docs/userGuides/Architecture/Cient/PROTOCOL-ERC-721.md]: Resolved.

**Review by TJ-Everett — APPROVED** [review]
Good to go after tests finish!

**Pull request merged**